### DISCUSSION TITLE
Purpose of vsomeip.lck file

### DISCUSSION THREAD

**Discussion by srchaitu2008** [COVESA/vsomeip#619] [Q&A] [answered]
Hi all,

I observed that when a vsomeip application is started, a file /tmp/vsomeip.lck is created with my username. Can anyone explain the purpose/significance of this file?

Thanks and Regards,
S.Ravi.Chaitanya

**Comment by duartenfonseca** [comment, marked as answer]
this ensures that only one routing manager is instantiated
